### ISSUE TITLE
Chat welcome message links lack sufficient differentiation

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#286587]
From an accessiblity perspective, links need to be distinguished by more than just color. We want links to have an underline on hover and focus like they do in the settings UI. 

Discovered via automated testing. 

https://github.com/user-attachments/assets/4c4d2f51-4288-47e2-9a9f-89103e253a14
